lib/rspec/core/memoized_helpers.rb

```diff
@@ -78,7 +78,7 @@ def subject
       # @note If you are using RSpec's newer expect-based syntax you may
       #       want to use `is_expected.to` instead of `should`.
       def should(matcher=nil, message=nil)
-        RSpec::Expectations::PositiveExpectationHandler.handle_matcher(subject, matcher, message)
+        is_expected.to(matcher, message)
```

Comment:
So I think the reason why this was the way it was, is because `expect` can be disabled

Reply:
I would also love to move `should`/`should_not`/`is_expected` to `rspec-expectations` in a follow-up. Can you think of some implications to that? All of them imply that:

> This only works if you are using rspec-expectations

Reply:
Might be now is a good moment to:
 - deprecate `should` with `OperatorMatcher`
 - retire `OperatorMatcher`

since `BeComparedTo` does this job anyway with better readability.

WDYT?

Reply:
With should being removed there is no need to deprecate a specific matcher with it.